test/membrane/instrumentation.spec.js

```diff
@@ -32,12 +32,12 @@ describe('Ensure instrumentation object usage in near-membrane', () => {
         expect(errorSpy).not.toHaveBeenCalled();
 
         redProxy.foo = 1;
-        expect(startActivitySpy).toHaveBeenCalledTimes(4);
```

Comment:
I missed this previously: why is the `startActivitySpy` called one less time?

Reply:
Because it's going through 1 less foreignCallable invocation (part of the optimization)